FOG/.htaccess

```diff
@@ -0,0 +1,29 @@
+Options +FollowSymLinks
+RewriteEngine on
+
+# Turn off MultiViews
+Options -MultiViews
+
+AddType text/turtle .ttl
+
+# In case of accept header <text/turtle>
+RewriteCond %{HTTP_ACCEPT} ^.*text/turtle.* 
+RewriteRule ^$ http://linkedbuildingdata.net/FOG/fogOnto.ttl [R=303,NE,L]
+
+# If suffix ttl, redirect to turtle version
+RewriteRule ^omg.ttl$ http://linkedbuildingdata.net/FOG/fogOnto.ttl [R=302,NE,L]
```

Comment:
Did you mean for the FOG sources to be omg.* or fogOnto.*?

Reply:
Thanks for pointin it out. It should be fog.*